squad-server/plugins/basic-stats.js

```diff
@@ -704,8 +697,6 @@ export default class DiscordStats extends BasePlugin {
         });
 
         const kdr = deaths === 0 ? kills : kills / deaths;
-        const rpd = revives === 0 ? 0 : revives / deaths;
```

Comment:
Same with these.

Reply:
There was no use for them anymore, Unless you would like me to add them to the leaderboard

Reply:
Re added them